db/seeds.rb

```diff
@@ -0,0 +1,45 @@
+require 'http'
+
```

Comment:
@pyreta nice work, you got the fixed width thing.

Overall comments: would love to see some specs for the behavior contained in this file. It would probably be easier to write those tests if you encapsulated the logic in `seeds.rb` into a class or classes.

So in `seeds.rb` you can just do something like `BankDataParser.perform!`, and the you'll have the actual (unit-tested) logic in `lib/<your-app-name>/bank_data_parser.rb`.